src/components/ClauseComponent.js

```diff
@@ -36,6 +44,70 @@ function ClauseComponent(props) {
     iconBg: clauseProps.CLAUSE_BACKGROUND
   };
 
+  /* eslint react/no-find-dom-node: "off" */
+  useEffect(() => {
+    const findPosition = (key) => {
+      const pathToConditional = props.editor.value.document.getPath(key);
+      const elementDOM = props.editor.findDOMNode(pathToConditional);
+      const popupPosition = 'bottom left';
+      const positionalStyle = {
+        popupStyle: { top: elementDOM.offsetTop, left: elementDOM.offsetLeft, transform: 'none' },
+        popupPosition,
+      };
+      return positionalStyle;
+    };
+
+    const findConditionals = node => ({
+      ...(node.type === 'conditional' ? {
+        [node.key]: {
+          whenTrue: node.data.get('whenTrue'),
+          whenFalse: node.data.get('whenFalse'),
+          position: findPosition(node.key),
+          isFalse: node.text === '',
+        }
+      } : {}),
+      ...(node.nodes || [])
+        .map(findConditionals)
+        .reduce((props, node) => ({ ...props, ...node }), {})
+    });
+
+    const newState = findConditionals(props.clauseNode);
+    setConditionals(newState);
+  }, [props.clauseNode, props.editor]);
+
+  const toggleConditional = (key) => {
+    const selectionNodeKey = key || props.editor.value.selection.focus.key - 1;
+    const selectedConditional = conditionals[selectionNodeKey];
+
+    if (selectedConditional) {
+      const selectionTextNode = props.editor.value.document
+        .getTextsAtRange(props.editor.value.selection)
+        .find(b => b.text);
+
+      const newInlineJSON = {
+        object: 'inline',
+        type: 'conditional',
+        data: {
+          id: 'forceMajeure',
+          whenTrue: selectedConditional.whenTrue,
+          whenFalse: selectedConditional.whenFalse
+        },
+        nodes: [
+          {
+            object: 'text',
+            text: selectionTextNode.text === selectedConditional.whenTrue
```

Comment:
Is that backward?

Reply:
I don't believe so. If the text is true, change it to false. If not, (it is false), change it to true.

Reply:
🤔 I'll take your word for it ! 😄